Workbooks/Network Security Perimeter/Access Logs/NSP Access Logs Insights.workbook

```diff
@@ -0,0 +1,98 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "74ba6ba7-94ee-4a5d-9752-a643b2512995",
+            "version": "KqlParameterItem/1.0",
+            "name": "FilterResource",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 2592000000
+            },
+            "value": ""
+          },
+          {
+            "id": "b06d4336-3f0d-44c7-adeb-b2902580ae18",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "label": "Time Range",
+            "type": 4,
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 43200000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 172800000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 1209600000
+                },
+                {
+                  "durationMs": 2419200000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                },
+                {
+                  "durationMs": 7776000000
+                }
+              ],
+              "allowCustom": true
+            },
+            "value": {
+              "durationMs": 2592000000
+            }
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.network/networksecurityperimeters"
+      },
+      "name": "parameters - 0"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "NSPAccessLogs \r\n| where ServiceResourceId contains \"{FilterResource}\"",
+        "size": 0,
+        "showAnalytics": true,
+        "title": "Access Logs for the selected paas resource",
+        "timeContextFromParameter": "TimeRange",
+        "showExportToExcel": true,
+        "queryType": 0,
+        "resourceType": "microsoft.network/networksecurityperimeters",
+        "gridSettings": {
```

Comment:
all my suggestions would be nits: like having this query return only meaningful columns, with nice renderers/etc configured instead of just defaults/etc.

if it is the only thing in the page, you could also make the size larger to take up more space?

could also possibly optimize using `has` instead of `contains` in the query, but that's more up to exactly what you're doing in the query and if that's better/worse for you.

Reply:
Good suggestions, I would have to get this page reviewed and take suggestions from PMs and users regarding columns so will update the column names in next update. I will look into has option more and update it later. Thanks @gardnerjr for all the support. Without your support this could not have been done.